src/main/java/carsharing/processors/impl/app/ManagerProcessor.java

```diff
@@ -0,0 +1,41 @@
+package carsharing.processors.impl.app;
+
+import carsharing.ConsoleReader;
+import carsharing.Constants;
+import carsharing.processors.ICompanyProcessors;
+import carsharing.processors.ICompanyProcessorsFactory;
+import carsharing.processors.IAppActionProcessor;
+
+public class ManagerProcessor implements IAppActionProcessor {
+
+    public ManagerProcessor(ICompanyProcessorsFactory companyProcessorsFactory) {
+        this.companyProcessorsFactory = companyProcessorsFactory;
+    }
+
+    private final ICompanyProcessorsFactory companyProcessorsFactory;
```

Comment:
please place constructor and field in a correct places

Reply:
Done